src/pages/SignUp.vue

```diff
@@ -1,14 +1,99 @@
 <template>
   <q-page padding>
-    <h1>Sign Up Page</h1>
+    <h1>Registration</h1>
```

Comment:
```suggestion
    <div class="text-h6 q-py-md">
      Sign Up
    </div>
```
Let's stick with "sign up" as terminology as that's what the buttons on the rest of the site say

Reply:
The classes used here will make the title match that on the Meme page and elsewhere